README.md

```diff
@@ -881,6 +890,7 @@ Since `script` is designed to help you write system administration programs, a f
 * [head](examples/head/main.go)
 * [least_freq](examples/least_freq/main.go)
 * [ls](examples/ls/main.go)
+* [prompt](examples/prompt/main.go)
```

Comment:
The shell builtin is called `read`, isn't it—perhaps we should use that name.

Reply:
(Shall we add this to the table of Unix equivalents as well, under `read`?)

Reply:
[_Nja_](https://en.wiktionary.org/wiki/nja#Swedish), while `read` is similar in function and behavior, it's not really exactly what `ReadInput` does here. For example `read DIR_NAME` doesn't directly pipe user input to the next command. It doesn't really have a built-in way for default input either.

Having said that, I agree that adding `read` to the list might help new comers use `ReadInput` so I suggest writing `read*` and explain the differences somehow.

Reply:
Well, that table is really to help people who are in the position of "I have some existing shell script, or something that I know how to do in shell, and I want to do it with `script`—what should I look for?" For example, if you want to read user input, you might reach for the `read` builtin, and we want you to be guided towards `ReadInput`, even though it's not an exact analogue.

Reply:
Done, please resolve.